### PR TITLE
Assigment 04 - Simon Chumacero Espada

### PR DESCRIPTION
**Pull request by SimonChumaceroEspada**

### CODE REVIEW

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -90,20 +90,13 @@
         <property name="message" value="Line has trailing spaces."/>
     </module>
 
-    <!-- Checks for Headers                                -->
```
Comment (marcos-x86) [inline config/checkstyle/checkstyle.xml]: Why do you modified this configurations?

File: src/main/java/bo/usfx/Example.java [hunk 2]
```diff
@@ -1,3 +1,7 @@
+/**
+ * Este paquete contiene la clase Example,que
```
Comment (marcos-x86) [inline src/main/java/bo/usfx/Example.java:2]: It's not necessary to add comments. But if you want to add them you have to change everything to English.

File: src/main/java/bo/usfx/exampletdd/package-info.java [hunk 3]
```diff
@@ -0,0 +1 @@
+package bo.usfx.exampletdd;
```
Comment (marcos-x86) [inline src/main/java/bo/usfx/exampletdd/package-info.java]: It's not necessary to add this file, please remove it.

File: src/main/java/bo/usfx/package-info.java [hunk 4]
```diff
@@ -0,0 +1 @@
+package bo.usfx;
```
Comment (marcos-x86) [inline src/main/java/bo/usfx/package-info.java]: It's not necessary to add this file, please remove it.

File: src/main/java/simon/chumacero/espada/assigment2/Kata1.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
+package simon.chumacero.espada.assigment2;
+
+public final class Kata1 {
+  /**
+   * Apartir de esta cantidad se considera una serie.
+   * */
+  private static final int SMELL_SERIE = 3;
```
Comment (marcos-x86) [inline src/main/java/simon/chumacero/espada/assigment2/Kata1.java]: The indentation level of this file is wrong, please reformat the code.

File: src/main/java/simon/chumacero/espada/assigment3/Kata1.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
+package simon.chumacero.espada.assigment3;
+
+public final class Kata1 {
```
Comment (marcos-x86) [inline src/main/java/simon/chumacero/espada/assigment3/Kata1.java]: The indentation level of this file is wrong, please reformat the code.

File: src/test/java/bo/usfx/exampletdd/WellTests.java [hunk 7]
```diff
@@ -7,9 +7,9 @@ public class WellTests {
 
     @Test
     public void simpleTest() {
-        Assert.assertEquals("Fail!", Well.well(new String[]{"bad", "bad", "bad"}));
-        Assert.assertEquals("Publish!", Well.well(new String[] {"good", "bad", "bad", "bad", "bad"}));
-        Assert.assertEquals("I smell a series!", Well.well(new String[] {
-                "good", "bad", "bad", "bad", "bad", "good", "bad", "bad", "good"}));
+        Assert.assertEquals(Well.well(new String[]{"bad", "bad", "bad"}), "Fail!");
```
Comment (marcos-x86) [inline src/test/java/bo/usfx/exampletdd/WellTests.java]: This change is already in the main branch, please update your branch everytime that you push new code.

File: src/test/java/simon/chumacero/espada/test/assigment3/test/assigment4/Kata1Test.java [hunk 8]
```diff
@@ -0,0 +1,31 @@
+package simon.chumacero.espada.test.assigment3.test.assigment4;
+
+import org.junit.Assert;
+import org.junit.Test;
+import simon.chumacero.espada.assigment4.Kata1;
+
+public class Kata1Test {
+
+  @Test
+  public void testRowWeights() {
```
Comment (marcos-x86) [inline src/test/java/simon/chumacero/espada/test/assigment3/test/assigment4/Kata1Test.java]: The indentation level of this file is wrong, please reformat the code.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

**Comment by SimonChumaceroEspada** [comment]
Thanks for the feedback

**Pull request closed without merging**